application/deed/deed-api.json

```diff
@@ -184,6 +185,10 @@
                 "md_ref": {
                     "pattern": "^e-MD([1-9]{5}|([1-9]{3,4}[A-Z]{1}))$",
                     "type": "string"
+                },
+                "identity_checked": {
+                  "pattern": "^[Y]$",
```

Comment:
This Regular expression only allows for Y. So are we saying that if they don't put a Y on creation on their deed application we'll reject it straight away? That seems incorrect as that check needs to only be done when either signing or making effective (need to check with BAs here)